### PR TITLE
Added a numpad keymap for the vitamins included keyboard

### PR DESCRIPTION
**Pull request by Duckle29**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/vitamins_included/keymaps/numpad/config.h [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef CONFIG_USER_H
```
Comment (drashna) [inline keyboards/vitamins_included/keymaps/numpad/config.h]: If you could replace the include guard (the ifndef, define, and endif at the end) with just `#pragma once`? 

It's simpler, and less prone to user error, so if you could make this switch? 

```suggestion
#pragma once
```

File: keyboards/vitamins_included/keymaps/numpad/config.h [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+#define CONFIG_USER_H
```
Comment (drashna) [inline keyboards/vitamins_included/keymaps/numpad/config.h]:
```suggestion
```

File: keyboards/vitamins_included/keymaps/numpad/config.h [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+#define EE_HANDS
+
+#ifdef AUDIO_ENABLE
+  #define DEFAULT_LAYER_SONGS { SONG(QWERTY_SOUND), \
+                                SONG(DVORAK_SOUND), \
+                                SONG(COLEMAK_SOUND) \
+                              }
+#endif
+
+#endif
```
Comment (drashna) [inline keyboards/vitamins_included/keymaps/numpad/config.h]:
```suggestion
```

File: keyboards/vitamins_included/keymaps/numpad/config.h [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef CONFIG_USER_H
+#define CONFIG_USER_H
+
+#include "../../config.h"
```
Comment (drashna) [inline keyboards/vitamins_included/keymaps/numpad/config.h]: This line is not actually needed, and should be removed. 

```suggestion
```

Specifically, the build script automatically includes all of the config.h files, so this is not necessary.

File: keyboards/vitamins_included/keymaps/numpad/keymap.c [hunk 5]
```diff
@@ -0,0 +1,81 @@
+#include QMK_KEYBOARD_H
+#include "action_layer.h"
```
Comment (drashna) [inline keyboards/vitamins_included/keymaps/numpad/keymap.c]: Could you remove all of these additional includes?  

```suggestion
```

They get included automatically, at various points in the built process, so they are redundant and not necessary here.

File: keyboards/vitamins_included/keymaps/numpad/keymap.c [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+#include "eeconfig.h"
```
Comment (drashna) [inline keyboards/vitamins_included/keymaps/numpad/keymap.c]:
```suggestion
```

File: keyboards/vitamins_included/keymaps/numpad/keymap.c [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+#define _NUMPAD 0
+#define _ADJUST 3
+
+enum custom_keycodes {
+  NUMPAD = SAFE_RANGE,
+  ADJUST
+};
+
+// Fillers to make layering more clear
+#define _______ KC_TRNS
```
Comment (drashna) [inline keyboards/vitamins_included/keymaps/numpad/keymap.c]: These defines aren't necessary, as these are actually defined in the core.  You can safely remove them. 

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L30-L31


```suggestion
```

File: keyboards/vitamins_included/keymaps/numpad/keymap.c [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+#define XXXXXXX KC_NO
```
Comment (fauxpark) [inline keyboards/vitamins_included/keymaps/numpad/keymap.c]: These two keycodes are already defined in QMK, there's no need to add them here :)

Comment (drashna) [inline keyboards/vitamins_included/keymaps/numpad/keymap.c]:
```suggestion
```

File: keyboards/vitamins_included/keymaps/numpad/rules.mk [hunk 9]
```diff
@@ -0,0 +1,3 @@
+ifndef QUANTUM_DIR
```
Comment (drashna) [inline keyboards/vitamins_included/keymaps/numpad/rules.mk]: Could you remove this (ifndef/include/endif) block? 

It's not needed, and is a holdover from the old build system.

File: keyboards/vitamins_included/keymaps/numpad/rules.mk [hunk 10]
```diff
@@ -0,0 +1,3 @@
...
+	include ../../../../Makefile
+endif
```
Comment (fauxpark) [inline keyboards/vitamins_included/keymaps/numpad/rules.mk]: This file can be deleted, the include does nothing.

**Comment by Duckle29** [comment]
I did the requested changes :) Things sure have changed a lot since I added the rev1 :)

**Comment by drashna** [comment]
@Duckle29 Yeah, definitely!  We have more people who are active, and keyboards/keymaps are getting merged a lot faster, too! 

Lots of cool stuff added, too.  

But thank you for taking the time to apply the suggestions!

**Pull request merged**